input/fsh/BFDR/bundlescompositions/CompositionProviderFetalDeathReportNew.fsh

```diff
@@ -0,0 +1,259 @@
+Profile: CompositionProviderFetalDeathReportNew
+Parent: Composition
+Title: "Composition - Provider Fetal Death Report"
+Description: "This Composition profile contains constraints to address the use case describing the need for fetal death information to be recorded and communicated to the jurisdictional Vital Records Office."
+* ^meta.versionId = "69"
+* ^meta.lastUpdated = "2023-04-25T05:58:09.576+00:00"
+* ^meta.source = "#q3kYsclRE3ANIm1f"
+* ^status = #active
+* ^experimental = false
+* ^publisher = "HL7 Public Health Work Group"
+* ^jurisdiction = urn:iso:std:iso:3166#US "United States of America"
+* extension 
+  * ^slicing.discriminator.type = #value
+  * ^slicing.discriminator.path = "url"
+  * ^slicing.rules = #open
+* extension contains
+    ExtensionFetalDeathReportNumber named fetalDeathReportNumber 0..1  and
+    ExtensionFetalDeathLocalFileNumber named fetalDeathLocalFileNumber 0..1  and
+    ExtensionDatereceivedByRegistrar named dateReceivedByRegistrar 1..1 and
+    ExtensionReplacementStatus named replacementStatus 0..1
+* extension[fetalDeathReportNumber] ^short = "State File Number"
+* extension[fetalDeathLocalFileNumber] ^short = "Local File No."
+// Status is deprecated (now flag in message header)
+* extension[replacementStatus] ^short = "Replace Status (deprecated)"
+* status 
+  * ^short = "In the case of a fetal death sent in error, a status of 'entered-in-error' must be set."
+  * ^definition = "In the case of a fetal death sent in error, a status of 'entered-in-error' must be set."
+* type = $loinc#69045-3
+* type 
+  * ^short = "U.S. standard report of fetal death - 2003 revision"
+  * ^definition = "U.S. standard report of fetal death - 2003 revision"
+* subject 1.. 
+* subject only Reference(PatientDecedentFetusVitalRecords)
+  * ^short = "The subject of the composition is the decedent fetus"
+  * ^definition = "The subject of the composition as a whole is the decedent fetus. Different sections in the composition have different focus (e.g.: the mother)."
+* encounter only Reference(Encounter_Maternity)
+* encounter 
+  * ^short = "The Encounter for the Composition is the mother's maternity encounter."
+  * ^definition = "The Encounter for the Composition is the mother's maternity encounter."
+* author ..1 
+* author only Reference(PractitionerVitalRecordsNew)
+  * ^short = "The author is the person who verifies/approves the accuracy of the data to be sent to the vital records system."
+  * ^definition = "Name and title of person completing the report. May be, but need not be, the same as the attendant at delivery."
+* section 1.. 
+  * ^slicing.discriminator.type = #pattern
+  * ^slicing.discriminator.path = "code"
+  * ^slicing.rules = #open
```

Comment:
I think this is the default for composition sections.   May be able to delete this.   Everywhere there are composition sections.